apps/prairielearn/src/tests/e2e/bulkInviteStudents.spec.ts

```diff
@@ -0,0 +1,182 @@
+import type { Page } from '@playwright/test';
+
+import { dangerousFullSystemAuthz } from '../../lib/authz-data-lib.js';
+import { features } from '../../lib/features/index.js';
+import { EXAMPLE_COURSE_PATH } from '../../lib/paths.js';
+import {
+  selectCourseInstanceById,
+  updateCourseInstanceModernPublishing,
+} from '../../models/course-instances.js';
+import { ensureUncheckedEnrollment, inviteStudentByUid } from '../../models/enrollment.js';
+import { syncCourse } from '../helperCourse.js';
+import { getOrCreateUser } from '../utils/auth.js';
+
+import { expect, test } from './fixtures.js';
+
+/**
+ * Waits for the job sequence page to show completion and checks for expected text in the job output.
+ */
+async function waitForJobAndCheckOutput(page: Page, expectedTexts: string[]) {
+  // Should be redirected to the job sequence page
+  await expect(page).toHaveURL(/\/jobSequence\//);
+
+  // Wait for job to complete (status badge shows Success)
+  await expect(page.locator('.badge', { hasText: 'Success' })).toBeVisible();
+
+  // Check for expected text in the job output (rendered in a <pre> element)
+  const jobOutput = page.locator('pre');
+  for (const text of expectedTexts) {
+    await expect(jobOutput.getByText(text)).toBeVisible();
+  }
+}
+
+interface TestUser {
+  uid: string;
+  name: string;
+}
```

Comment:
We already have an `AuthUser` type that's compatible with helpers like `getorCreateUser(...)`; why make a new one?